themes/bootstrap5/templates/search/list-list.phtml

```diff
@@ -5,6 +5,8 @@
   $i = $this->indexStart;
   $listStart = $this->results->getStartRecord() + $i - $this->indexStart;
   $showCheckboxes = $this->searchSettings($this->results->getParams())->checkboxesEnabled();
+  $between = $this->results->getRecommendations('between');
+  $betweenPlacements = $this->between()->getPlacements($between, $this->results);
```

Comment:
As an alternative to creating a full `between` helper, I wonder if it's worth considering adding a public `getBetweenPlacement` method to the existing `recommend` helper. This could be achieved by refactoring the current `__invoke` method to return the whole object if `$recommend` is null and adding additional public methods.

(I'm also open to keeping the new helper if that's preferable -- especially if you expect it to get more complicated... but it seems like adding to the existing helper might have better semantics).